src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/DocumentController.kt

```diff
@@ -210,14 +243,19 @@ data class UpdateDocumentResponse(
   val details: String? = null,
 )
 
-data class AddDocumentRequest(
+data class UploadDocumentRequest(
   val category: DocumentCategory,
   val fileContent: String,
   val fileName: String,
   val details: String? = null
 )
 
-data class AddDocumentResponse(val documentId: DocumentId)
+data class GenerateDocumentRequest(
+  val category: DocumentCategory,
+  val details: String
+)
+
+data class NewDocumentResponse(val documentId: DocumentId)
```

Comment:
Not a fan of this name but we can change later - when we think of a better name.